.gitignore

```diff
@@ -371,3 +371,7 @@ android/pytorch_android_torchvision/.cxx
 # Pyre configs (for internal usage)
 .pyre_configuration
 .pyre_configuration.codenav
+.arcconfig
+.watchmanconfig
```

Comment:
There is already an entry for `.watchmanconfig` above, so you don't need to add it here again